CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Constellation Changes
 
+## Changes in July 2021
+
+-   Exceptions thrown in the `DefaultPluginEnvironment` are now presented to 
```

Comment:
Are you able to re-words this entry to start with the word "Updated"? Just to align it with other changelog entries